traffic_ops/client/deliveryservice.go

```diff
@@ -394,6 +394,16 @@ func (to *Session) GetDeliveryServiceCapacity(id string) (*tc.DeliveryServiceCap
 
 // GetDeliveryServiceServer returns associations between Delivery Services and servers using the
 // provided pagination controls.
+func (to *Session) GetDeliveryServiceRouting(id string) (*tc.DeliveryServiceRouting, ReqInf, error) {
```

Comment:
So this route does exist in TO API v2.0, but the client method doesn't exist in method, and as written it won't work. There's nothing wrong with leaving it in - but it needs to be fixed. Basically just add an API endpoint above and use it here instead of `deliveryServiceRoutingEp(id)` - as that function no longer exists.

Reply:
Oh, also the GoDoc above this `func` actually belongs to the client method below: `GetDeliveryServiceServer`.